ross/disk_element.py

```diff
@@ -555,3 +555,562 @@ def disk_example():
     """
     disk = DiskElement(0, 32.589_727_65, 0.178_089_28, 0.329_563_62)
     return disk
+
+
+class DiskElement6DoF(Element):
```

Comment:
Hi @raphaeltimbo , by default we left the plot functions and others present, in case changes proved necessary, but here they really are not. I will remove them and inherit the DiskElement class. Thanks for the tip!